app/services/work_packages/create_service.rb

```diff
@@ -67,25 +67,25 @@ def create(attributes, work_package)
       end
 
       set_user_as_watcher(work_package)
-    else
-      result.success = false
     end
 
     result
   end
 
-  def set_attributes(attributes, wp)
-    attributes_service_class
-      .new(user:,
-           model: wp,
-           contract_class:)
-      .call(attributes)
+  def set_templated_subject(work_package)
+    return true unless work_package.type&.replacement_patterns_defined?
```

Comment:
We're doing this a few times. Does it make sense to add a helper in the work_package for syntax sugar?

```ruby
work_package.type&.replacement_patterns_defined?
work_package.replacement_patterns_defined?
```

... probably no :sweat_smile:

Reply:
I wondered the same thing. Ended up not doing it... for now. :P